dev_requirements.txt

```diff
@@ -13,6 +13,7 @@ black~=19.10b0
 mypy~=0.910
 
 # Documentation and examples.
+jupytext
```

Comment:
This is the only change that could be related to readthedocs failng.

We could try to pin it and maybe move at at the bottom of the list (I don't know if the order could be important during the install process).